### PR TITLE
Add option to show trigger only once and then resume automatic load.

### PR DESCRIPTION
**Pull request by jblandry**

### CODE REVIEW

File: src/extension/trigger.js [hunk 1]
```diff
@@ -28,7 +30,7 @@ var IASTriggerExtension = function(options) {
       return true;
     }
 
-    if (false === this.offset || ++this.count < this.offset) {
+    if (false === this.offset || ++this.count < this.offset || (true === this.shown && null === this.$triggerNext)) {
```
Comment (fieg) [inline src/extension/trigger.js]: Why the `&& null === this.$triggerNext` condition?

File: src/extension/trigger.js [hunk 2]
```diff
@@ -17,6 +17,8 @@ var IASTriggerExtension = function(options) {
   this.enabled = true;
   this.count = 0;
   this.offset = options.offset;
+  this.showOnce = !!options.showOnce;
```
Comment (fieg) [inline src/extension/trigger.js]: `this.showOnce` is never checked?

**Comment by fieg** [comment]
Also this doesn't seem to pass the tests.

File: src/extension/trigger.js (hunk 2 above)
Comment (jblandry) [inline src/extension/trigger.js]: Oops forgot to use the flag corrected by 8bcaaac

**Comment by jblandry** [comment]
8bcaaac fixed the tests, only timeouts remains :)

**Comment by fieg** [comment]
Hey, sorry I didn't get back to you any time sooner. You PR looks good now. Tests are also passing, great! 

I'm still questioning the condition I commented earlier. I believe with your latest commit it's no longer needed?

**Comment by jblandry** [comment]
If you're refering to https://github.com/webcreate/infinite-ajax-scroll/pull/246#discussion_r67651951 I explained it here : https://github.com/webcreate/infinite-ajax-scroll/pull/246#discussion_r68651178

If you're refering to https://github.com/webcreate/infinite-ajax-scroll/pull/246#discussion-diff-68308816 I wasn't using the flag which was corrected by my second commit.

**Comment by fieg** [comment]
This PR is closed because Infinite Ajax Scroll 2 [reached end of life](https://infiniteajaxscroll.com/blog/end-of-life-ias2/) and is being [superseded by v3](https://infiniteajaxscroll.com).

@jblandry I would like to apologize that I didn't follow up on your PR and your hard work.

If you have questions, feedback or think this PR was closed by mistake, please comment on this PR.

**Comment by jblandry** [comment]
No problem, we had to make the change on our side anyway. :)

**Pull request closed without merging**